### DISCUSSION TITLE
VDI CodING - Hall of Fame

### DISCUSSION THREAD

**Discussion by stan1025** [VDI-CodING/codeING-main#21] [General]
Wenn ihr an den Coding Dojos der VDI Young Engineers teilnehmt, hinterlasst doch hier mal ein Post, dass ihr Teilnehmer seid!

**Comment by w-tusche** [comment]
Moin, 
mich habt Ihr noch nicht abhängen können. Bin noch mit Spaß dabei.

Vielen Dank für die Mühen!

**Reply by su-ro** [reply]
Ich bin die Susi und ich bin auch noch dabei

**Reply by kogedi** [reply]
Ich bin dabei!  👍
